plasmapy/utils/units_helpers.py

```diff
@@ -0,0 +1,75 @@
+"""Helper functionality for `astropy.units`."""
+
+from __future__ import annotations
+
+__all__ = ["get_physical_type_dict"]
+
+import astropy.units as u
+
+from numbers import Number
+from typing import Dict, Iterable
+
+
+def get_physical_type_dict(
```

Comment:
I think I'd indeed prefer for it to be private, at least for now. It's always easier to turn things from private to public than the other way around.